suite-native/atoms/src/Button/Button.tsx

```diff
@@ -1,118 +1,202 @@
-import React from 'react';
-import { TouchableOpacity, TouchableOpacityProps, View } from 'react-native';
+import React, { useState } from 'react';
+import { Pressable, PressableProps, View } from 'react-native';
+import Animated from 'react-native-reanimated';
 
-import { Color } from '@trezor/theme';
+import { MergeExclusive } from 'type-fest';
+
+import { Color, TypographyStyle, nativeSpacings } from '@trezor/theme';
 import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Icon, IconName } from '@trezor/icons';
 
 import { Text } from '../Text';
+import { useButtonPressAnimatedStyle } from './useButtonPressAnimatedStyle';
 
 export type ButtonSize = 'small' | 'medium' | 'large';
-export type ButtonColorScheme = 'primary' | 'gray' | 'red';
+export type ButtonColorSchemeName = 'primary' | 'secondary' | 'tertiary' | 'danger';
 
-export interface ButtonProps extends Omit<TouchableOpacityProps, 'style'> {
-    iconName?: IconName;
-    colorScheme?: ButtonColorScheme;
+export type ButtonProps = Omit<PressableProps, 'style'> & {
+    children: string;
+    colorSchemeName?: ButtonColorSchemeName;
```

Comment:
can you drop `Name`from prop? it's unneccesary long with no added information value

Reply:
fixed in ddbfd1430